src/runtime/GlobalObjectBuiltinRegExp.cpp

```diff
@@ -359,6 +359,67 @@ GlobalRegExpFunctionObject::GlobalRegExpFunctionObject(ExecutionState& state)
     initInternalProperties(state);
 }
 
+#ifdef ESCARGOT_ENABLE_ES2015
+
+static Value builtinRegExpOptionGetterHelper(ExecutionState& state, Value thisValue, unsigned int option)
+{
+    if (!thisValue.isObject() || !thisValue.asObject()->isRegExpObject(state)) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::Code::TypeError, "getter called on non-RegExp object");
+    }
+
+    // ES2017 compatibility
+    if (thisValue.asObject() == state.context()->globalObject()->regexpPrototype()) {
+        return Value();
+    }
+
+    return Value((bool)(thisValue.asObject()->asRegExpObject(state)->option() & option));
+}
+
+static Value builtinRegExpFlagsGetter(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+{
```

Comment:
ASSERT(argv != nullptr)

Reply:
I don't think that's necessary, since argv is not at all used in this function, and the callee passes a nullptr as default, with 0 arguments given. Maybe we could do something like 
```
if (argc > 0) {
  ASSERT(argv != nullptr);
}
```

Then again, this function doesn't use the argv parameter.